doc/man/man3/crypto_x25519.3monocypher

```diff
@@ -106,6 +106,12 @@ If either of the long term secret keys leaks, it may compromise
 .Em all past messages .
 This can be avoided by using protocols that provide forward secrecy,
 such as the X3DH key agreement protocol.
```

Comment:
I'll have to reflect on whether I want to keep this one or not. X3DH only makes sense in a Signal-like infrastructure. I'd rather mention Noise, Monokex (when ready, or just what you added below).